gdx/src/com/badlogic/gdx/assets/AssetManager.java

```diff
@@ -572,6 +574,7 @@ public synchronized void setErrorListener (AssetErrorListener listener) {
 	}
 
 	/** Disposes all assets in the manager and stops all asynchronous loading. */
+	@Override
```

Comment:
Please do not add `Override` where it's missing, I think @NathanSweet is not too fond of that practise.